api/model_enums/__init__.py

```diff
@@ -22,4 +30,5 @@ def create_enum_app():
 def create_enum_db(enum_app):
 	enum_db = SQLAlchemy()
 	enum_db.init_app(enum_app)
+
 	return enum_db
```

Comment:
Looks like this file is missing a new line at the end -- also looks like it wasn't you that caused it!